modules/deployments-ab-testing-lb.adoc

```diff
@@ -205,7 +205,7 @@ Not all routers may support multiple or weighted backends.
 +
 [source,terminal]
 ----
-$ oc new-app openshift/deployment-example --name=ab-example-a
+$ oc new-app openshift/deployment-example --name=ab-example-a --labels=ab-example=true
```

Comment:
The `oc new-app` default create deploy instead of DC， so the command should be :   
`oc new-app  openshift/deployment-example --name=ab-example-a --as-deployment-config=true --labels=ab-example\=true --env=SUBTITLE\=shardA`

Reply:
As before , the second App, should also specify created as DC:
`oc new-app openshift/deployment-example:v2  --name=ab-example-b --labels=ab-example=true  SUBTITLE="shard B" COLOR="red"  --as-deployment-config=true`